### DISCUSSION TITLE
What do you use for TODOs?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#1384] [Q&A] [answered]
This is off-topic and it's not a question for PocketBase.

I'm just curious since several times I've read in many discussions @ganigeorgiev  mentioning "_it is in my todo_".
Is this just a figure of speech or you are using an actual TODO app?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm using an actual todo/task app. Well, technically it is not an app, but a Sublime plugin - [PlainTasks](https://packagecontrol.io/packages/PlainTasks).

Here is a screenshot if you are curious how it looks (_ignore the font-size_):
![screenshot](https://i.imgur.com/ZxwRUYD.png)

But this is just my personal reminder todo/tasks list. For the actual tasks that are currently in progress please refer to the [roadmap](https://github.com/orgs/pocketbase/projects/2).

**Reply by denisgurev** [reply]
Thank you for the prompt reply!

This is very clever, I never thought before searching for an editor plugin for TODOs. I'm using vscode and just found a similar plugin https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-todo-plus#demo.
